Automattic/wp-calypso
# Blog onboarding: Update celebration screen for mobile devices

Related to 2556-gh-Automattic/dotcom-forge

## Proposed Changes

* Improved mobile view of celebration screen

| Before | After |
|--------|--------|
| <img width="489" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1044309/17f53e79-aeb3-4633-ab58-b567185242ee"> | <img width="486" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1044309/10b17163-b428-44b4-ab00-9c9678595533"> | 

## Testing Instructions



* Load https://github.com/Automattic/jetpack/pull/30871 to your WPCOM Sandbox, but since the Jetpack PR is now merged, if you can't load it with the command above, you can use this diff instead D111566-code
* Go to /setup/design-first on an account with 0 existing sites.
* Complete the process and you should see the celebration screen
* Replace design-first for start-writing to test both flows (it's the same implementation)
* It should continue centered on non small devices

## Pre-merge Checklist
